File: src/main/java/com/movisens/xs/api/exceptions/ServerException.java
package com.movisens.xs.api.exceptions;

import com.movisens.xs.api.models.ApiError;

public class ServerException extends MovisensXSException {

    private static final long serialVersionUID = 5596568349506604008L;

    public ServerException(ApiError apiError) {
        super(apiError);
    }
}

File: src/main/java/com/movisens/xs/api/exceptions/ServerException.kt
package com.movisens.xs.api.exceptions

import com.movisens.xs.api.models.ApiError

class ServerException(apiError: ApiError?) : MovisensXSException(apiError) {
    companion object {
        private const val serialVersionUID = 5596568349506604008L
    }
}